test/conformance/CMakeLists.txt

```diff
@@ -18,10 +29,27 @@ function(add_conformance_test name)
         ${PROJECT_NAME}::testing
         ${PROJECT_NAME}::common
         GTest::gtest_main)
-    add_test(NAME ${name}
-        COMMAND ${TEST_TARGET_NAME}
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
-    set_tests_properties(${name} PROPERTIES LABELS "conformance")
+
+    if(${USE_LOCAL_TEST_CONFIGURATION})
```

Comment:
in `if` you can skip `${}` syntax (applies here and a few lines below)

Reply:
Done.